src/rpdk/core/contract/resource_client.py

```diff
@@ -587,6 +587,8 @@ def _call(self, payload):
     def call_and_assert(
         self, action, assert_status, current_model, previous_model=None, **kwargs
     ):
+        if not self.has_required_handlers_exist():
+            raise ValueError("Create/Read/Delete/List handlers are required")
```

Comment:
The error message says List is required but the method does not enforce it?

Reply:
My bad, I missed it. I was thinking of asserting list but decided against it